### ISSUE TITLE
[v0.9.0-rc2] Unnecessary print and unclear client version

### ISSUE THREAD

**Issue by warmchang** [rancher/k3s#786]
**Describe the bug**
Unnecessary print and unclear client version.

**To Reproduce**
Run "k3s --version" & "k3s kubectl version":
```
 root@william  /home/fedora  k3s --version                                                                                                       ✔  ⚡  8028  08:57:14
k3s version v0.9.0-rc2 (4a5360ea)
 root@william  /home/fedora                                                                                                                      ✔  ⚡  8028  08:57:17
 root@william  /home/fedora  k3s kubectl version                                                                                                     ✔  ⚡  8028  08:57:17
I0902 08:57:20.331650   17965 openapi.go:21] Instantiated OpenAPI definitions in 178.20416ms
Client Version: version.Info{Major:"", Minor:"", GitVersion:"v0.0.0-master+$Format:%h$", GitCommit:"$Format:%H$", GitTreeState:"", BuildDate:"1970-01-01T00:00:00Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3-k3s.1", GitCommit:"77c93d8d16059eb6873a8a154bbd70518b4e9db4", GitTreeState:"clean", BuildDate:"2019-08-30T18:26+00:00Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
 root@william  /home/fedora                                                                                                                      ✔  ⚡  8029  08:57:20
```

**Expected behavior**

1. Don't print the info: "I0902 08:57:20.331650   17965 openapi.go:21] Instantiated OpenAPI definitions in 178.20416ms"
2. Correct the Client Version.

**Screenshots**
![image](https://user-images.githubusercontent.com/13084624/64103246-5d96b800-cda4-11e9-8a21-8aca7b3390bc.png)

**Additional context**
manually upgrade from v0.8.1: copy k3s to "/usr/local/bin/k3s" and run "systemctl restart k3s".

**Comment by warmchang** [comment]
1, Look like a print info for debugging:
https://github.com/rancher/k3s/blob/8a9c8675c2a16591e13a655617acdb1e75d3962b/vendor/k8s.io/kubernetes/pkg/generated/openapi/openapi.go#L21

2, Overwrite the file "https://github.com/rancher/k3s/blob/v1.15.3-k3s.1/pkg/kubectl/version/base.go" with the following content, and remake the v1.15.3-k3s.1 git tag.

```
package version

var (
	gitMajor = "1"
	gitMinor = "15"
	gitVersion   = "v1.15.3-k3s.1"
	gitCommit    = "77c93d8d16059eb6873a8a154bbd70518b4e9db4"
	gitTreeState = "clean"
	buildDate = "2019-08-30T18:26+00:00Z"
)
```

or make a new tag (maybe v1.15.3-k3s.2), and upgrade the tag in vendor.conf:
https://github.com/rancher/k3s/blob/8031137b794306a07de2f94a01ae189dfcafe1e7/vendor.conf#L12

**Comment by ibuildthecloud** [comment]
Oh shoot, yeah.  This was just supposed to print on server startup but it gets invoked on all startup.  I'll move this to V(7) or something.

**Comment by warmchang** [comment]
It has been fixed in version [v0.9.0](https://github.com/rancher/k3s/releases/tag/v0.9.0).


```
 root@william  /home/fedora  k3s --version                                                                                                       ✔  ⚡  8749  04:21:52
k3s version v0.9.0 (65d87648)
 root@william  /home/fedora  k3s kubectl version                                                                                                 ✔  ⚡  8749  04:21:52
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.4-k3s.1", GitCommit:"a7531b1ab3fd5ff987b074472ddfb84a2f5326bc", GitTreeState:"clean", BuildDate:"2019-09-19T22:36Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.4-k3s.1", GitCommit:"a7531b1ab3fd5ff987b074472ddfb84a2f5326bc", GitTreeState:"clean", BuildDate:"2019-09-19T22:36Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
 root@william  /home/fedora                                                                                                                      ✔  ⚡  8750  04:22:01

```
